rest/kadai-rest-spring/src/test/java/io/kadai/task/rest/TaskControllerIntTest.java

```diff
@@ -1270,11 +1272,38 @@ void should_GetAllTasks_For_GettingLastTaskSummaryPageSortedByPorValue() {
           .contains("page=16");
       assertThat(response.getBody().getContent().iterator().next().getTaskId())
           .isEqualTo("TKI:000000000000000000000000000000000064");
+
```

Comment:
I don't think we need this part. Asserting the first or last item should be enough.

Reply:
Removed it